exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,9 @@
 // Start by creating a variable `message`
+let myName="Abdulaziz";
+let length=myName.length;
+let intro= "My name is "+myName+" and my name is "+length+" characters long"
+
+
+
+console.log(intro);
```

Comment:
Hi Abdulaziz, well done to your exercise. Just a quick one, perhaps there is too much gap between your lines.

Reply:
Thank you Domenico